systemd.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Spacelift worker
+
+[Service]
```

Comment:
I would create separate one-time service to pull tags-metadata, token and secrets.

Reply:
token is passed as userdata into /etc/sysconfig/spacelift-launcher

Reply:
in terms of one-time service, let's add it in next iteration?